packages/fe-mockserver-plugin-cds/src/index.ts

```diff
@@ -26,6 +27,29 @@ export default class CDSMetadataProvider implements IMetadataProcessor {
      */
     constructor(private fileLoader: IFileLoader, private options?: CDSMetadataProviderOptions) {}
 
+    async loadI18nMapFolder(targetFolder: string, i18nMap: Record<string, string>): Promise<void> {
+        if (await this.fileLoader.exists(targetFolder)) {
+            const i18nProp = await this.fileLoader.loadFile(targetFolder + 'i18n.properties');
+            const i18nPropLines = i18nProp.split('\n');
+            for (const line of i18nPropLines) {
+                const [key, value] = line.split('=');
```

Comment:
Not 100% sure, so better leave it as it is now: I think something like this edge case
```
key=This does not work if X=5
```
is valid in terms of .properties files, but would not work here.

Reply:
You can use a regex in the `split`. Something like `const [key, value] = line.split(/=(.*)/s);` should do. The array might contain more than two entries but the others are empty and can be ignored.

Reply:
done